lib/matplotlib/mathtext.py

```diff
@@ -407,51 +407,48 @@ def destroy(self):
 
     def get_kern(self, font1, fontclass1, sym1, fontsize1,
                  font2, fontclass2, sym2, fontsize2, dpi):
-        r"""
+        """
         Get the kerning distance for font between *sym1* and *sym2*.
 
-        *fontX*: one of the TeX font names::
-
-          tt, it, rm, cal, sf, bf or default/regular (non-math)
-
-        *fontclassX*: TODO
-
-        *symX*: a symbol in raw TeX form. e.g., '1', 'x' or '\sigma'
-
-        *fontsizeX*: the fontsize in points
-
-        *dpi*: the current dots-per-inch
+        See `get_metrics` for a detailed description of the parameters.
```

Comment:
~~~
/home/circleci/project/lib/matplotlib/mathtext.py:docstring of matplotlib.mathtext.StandardPsFonts.get_kern:4: WARNING: py:obj reference target not found: get_metrics
/home/circleci/project/lib/matplotlib/mathtext.py:docstring of matplotlib.mathtext.TruetypeFonts.get_kern:4: WARNING: py:obj reference target not found: get_metrics
~~~
likely due to inherited docstrings. Maybe:

```suggestion
        See `~.Font.get_metrics` for a detailed description of the parameters.
```

Reply:
sure